baselines/dom.generated.d.ts

```diff
@@ -1819,6 +1819,7 @@ declare var CryptoKeyPair: {
 };
 
 interface CSS {
+    escape(value?: string): string;
```

Comment:
to be honest I am not sure 100% that the value should be a `string` as while the input is a `DOMString` from the example it seems that It can accept a number.
Example
```ts
CSS.escape(".foo#bar")        // "\.foo\#bar"
CSS.escape("()[]{}")          // "\(\)\[\]\{\}"
CSS.escape('--a')             // "--a"
CSS.escape(0)                 // "\30 ", the Unicode code point of '0' is 30
CSS.escape('\0')              // "\ufffd", the Unicode REPLACEMENT CHARACTER
```